code/modules/vore/fluffstuff/custom_clothes_vr.dm

```diff
@@ -1316,3 +1316,277 @@ Departamental Swimsuits, for general use
+	name = "Black and light grey cloak"
+	icon_state = "tesh_cloak_blg"
+	item_state = "tesh_cloak_blg"
+
+/obj/item/clothing/suit/storage/seromi/cloak/black_white
+	name = "Black and white cloak"
+	icon_state = "tesh_cloak_bw"
+	item_state = "tesh_cloak_bw"
+
+/obj/item/clothing/suit/storage/seromi/cloak/black_red
+	name = "Black and red cloak"
+	icon_state = "tesh_cloak_br"
+	item_state = "tesh_cloak_br"
+
+/obj/item/clothing/suit/storage/seromi/cloak/black
+	name = "Black cloak"
+	icon_state = "tesh_cloak_bn"
+	item_state = "tesh_cloak_bn"
+
+/obj/item/clothing/suit/storage/seromi/cloak/black_yellow
+	name = "Black and yellow cloak"
+	icon_state = "tesh_cloak_by"
+	item_state = "tesh_cloak_by"
+
+/obj/item/clothing/suit/storage/seromi/cloak/black_green
+	name = "Black and Green cloak"
+	icon_state = "tesh_cloak_bgr"
+	item_state = "tesh_cloak_bgr"
+
+/obj/item/clothing/suit/storage/seromi/cloak/black_blue
+	name = "Black and blue cloak"
+	icon_state = "tesh_cloak_bbl"
+	item_state = "tesh_cloak_bbl"
+
+/obj/item/clothing/suit/storage/seromi/cloak/black_purple
+	name = "Black and purple cloak"
+	icon_state = "tesh_cloak_bp"
+	item_state = "tesh_cloak_bp"
+
+/obj/item/clothing/suit/storage/seromi/cloak/black_pink
+	name = "Black and pink cloak"
+	icon_state = "tesh_cloak_bpi"
+	item_state = "tesh_cloak_bpi"
+
+/obj/item/clothing/suit/storage/seromi/cloak/black_brown
+	name = "Black and brown cloak"
+	icon_state = "tesh_cloak_bbr"
+	item_state = "tesh_cloak_bbr"
+
+/obj/item/clothing/suit/storage/seromi/cloak/orange_grey
+	name = "Orange and grey cloak"
+	icon_state = "tesh_cloak_og"
+	item_state = "tesh_cloak_og"
+
+/obj/item/clothing/suit/storage/seromi/cloak/rainbow
+	name = "Rainbow cloak"
+	icon_state = "tesh_cloak_rainbow"
+	item_state = "tesh_cloak_rainbow"
+
+/obj/item/clothing/suit/storage/seromi/cloak/lightgrey_grey
+	name = "Light grey and grey cloak"
+	icon_state = "tesh_cloak_lgg"
+	item_state = "tesh_cloak_lgg"
+
+/obj/item/clothing/suit/storage/seromi/cloak/white_grey
+	name = "White and grey cloak"
+	icon_state = "tesh_cloak_wg"
+	item_state = "tesh_cloak_wg"
+
+/obj/item/clothing/suit/storage/seromi/cloak/red_grey
+	name = "Red and grey cloak"
+	icon_state = "tesh_cloak_rg"
+	item_state = "tesh_cloak_rg"
+
+/obj/item/clothing/suit/storage/seromi/cloak/orange
+	name = "Orange cloak"
+	icon_state = "tesh_cloak_on"
+	item_state = "tesh_cloak_on"
+
+/obj/item/clothing/suit/storage/seromi/cloak/yellow_grey
+	name = "yellow and grey cloak"
+	icon_state = "tesh_cloak_yg"
+	item_state = "tesh_cloak_yg"
+
+/obj/item/clothing/suit/storage/seromi/cloak/green_grey
+	name = "Green and grey cloak"
+	icon_state = "tesh_cloak_gg"
+	item_state = "tesh_cloak_gg"
+
+/obj/item/clothing/suit/storage/seromi/cloak/blue_grey
+	name = "Blue and grey cloak"
+	icon_state = "tesh_cloak_blg"
+	item_state = "tesh_cloak_blg"
+
+/obj/item/clothing/suit/storage/seromi/cloak/purple_grey
+	name = "purple and grey cloak"
```

Comment:
Non-unique items should never start with a capital letter in the name, unless it's preceded by `\improper` because otherwise when you examine it, it looks stupid.

Right now this will appear as: `That's a purple and grey cloak`
If you capitalize it, it will be: `That's Purple and grey cloak`